packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -18,21 +18,48 @@ import { assert, performance } from "@fluidframework/common-utils";
  */
 export const latencyThreshold = 5000;
 
+// Stages in OpPerfTelemetry that fire in the following oder:
+// 1) "submitOp" event
+// 2) DeltaManager's outbound "push" event
+// 3) DeltaManager's inbound "push" event
+// 4) Start of op processing (no explicit event for)
+// 5) DeltaManager "op" event(time to process an op)
```

Comment:
It is fine as it is (i.e. feel free to leave it as you have it), but I'll share how I think :)
I'd probably describe stages as long processes. I.e.
Stages:

1. Op sits in buffer in DeltaManager queue, then in outbound queue for some time
   - We do not differentiate these two today in telemetry, but first one is due to batches, second one might happen due to outbound queue being paused
2. Op is sent to service and back
3. op sits in inbound queue
4. Op is processed.

I.e. events follow (not the primary way to describe processes), and events fire in between phases (i.e. they clearly denote the boundary between phases).